OpenCensusAgentPluggin/opencensus-service/exporter/loggingexporter/logging_exporter_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019, OpenCensus Authors
```

Comment:
it doesn't seem like we need these tests, can you exclude these too?

Reply:
Yeah, I wasn't sure if I should take out all the tests. That is something I can definitely do.